azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -1,5 +1,7 @@
 import sys
 from typing import TYPE_CHECKING, Union
+import numpy as np
```

Comment:
Remove these - before this we were careful to not just import these modules, since (for now) the pandas feature is optional, and users might not have these packages.